control-plane/pkg/reconciler/channel/resources/service.go

```diff
@@ -62,7 +68,7 @@ func ExternalService(namespace, service string) ServiceOption {
 // MakeK8sService creates a new K8s Service for a Channel resource. It also sets the appropriate
 // OwnerReferences on the resource so handleObject can discover the Channel resource that 'owns' it.
 // As well as being garbage collected when the Channel is deleted.
-func MakeK8sService(kc *v1beta1.KafkaChannel, opts ...ServiceOption) (*corev1.Service, error) {
+func MakeK8sService(kc *v1beta1.KafkaChannel, logger *zap.Logger, opts ...ServiceOption) (*corev1.Service, error) {
```

Comment:
@Leo6Leo do you need the logger in this function? I'm not seeing it called anywhere below
```suggestion
func MakeK8sService(kc *v1beta1.KafkaChannel, opts ...ServiceOption) (*corev1.Service, error) {
```

Reply:
No need. Haven't cleaned up the code yet.

Reply:
@Leo6Leo I think this is part of why some are failing:
```
control-plane/pkg/reconciler/channel/v2/channelv2.go:340:53: cannot use resources.ExternalService(r.DataPlaneNamespace, channelreconciler.NewChannelIngressServiceName) (value of type resources.ServiceOption) as *zap.Logger value in argument to resources.MakeK8sService
```